t/errlog.t

```diff
@@ -932,3 +932,65 @@ maybe log lines: #1
 end
 \z
 --- skip_nginx: 2: <1.11.2
+
+
+
+=== TEST 23: the system default filter level is "debug"
```

Comment:
@membphis 
Yes, this branch is already covered with `TEST 25: get system default filter level during Nginx worker starts`.